packages/patient-portal/src/views/SurveyView.tsx

```diff
@@ -1,81 +1,61 @@
-  [PROGRAM_DATA_ELEMENT_TYPES.AUTOCOMPLETE]: PlaceholderField,
-  [PROGRAM_DATA_ELEMENT_TYPES.DATE]: PlaceholderField,
-  [PROGRAM_DATA_ELEMENT_TYPES.DATE_TIME]: PlaceholderField,
-  [PROGRAM_DATA_ELEMENT_TYPES.SUBMISSION_DATE]: PlaceholderField,
-  [PROGRAM_DATA_ELEMENT_TYPES.NUMBER]: PlaceholderField,
-  [PROGRAM_DATA_ELEMENT_TYPES.BINARY]: PlaceholderField,
-  [PROGRAM_DATA_ELEMENT_TYPES.CHECKBOX]: PlaceholderField,
-  [PROGRAM_DATA_ELEMENT_TYPES.CALCULATED]: ReadOnlyTextField,
-  [PROGRAM_DATA_ELEMENT_TYPES.SURVEY_LINK]: PlaceholderField,
-  [PROGRAM_DATA_ELEMENT_TYPES.SURVEY_RESULT]: null,
-  [PROGRAM_DATA_ELEMENT_TYPES.SURVEY_ANSWER]: PlaceholderField,
-  [PROGRAM_DATA_ELEMENT_TYPES.PATIENT_DATA]: ReadOnlyTextField,
-  [PROGRAM_DATA_ELEMENT_TYPES.USER_DATA]: ReadOnlyTextField,
-  [PROGRAM_DATA_ELEMENT_TYPES.INSTRUCTION]: PlaceholderField,
-  [PROGRAM_DATA_ELEMENT_TYPES.PHOTO]: PlaceholderField,
-  [PROGRAM_DATA_ELEMENT_TYPES.RESULT]: null,
-  [PROGRAM_DATA_ELEMENT_TYPES.PATIENT_ISSUE]: PlaceholderField,
-  [PROGRAM_DATA_ELEMENT_TYPES.COMPLEX_CHART_INSTANCE_NAME]: PlaceholderField,
-  [PROGRAM_DATA_ELEMENT_TYPES.COMPLEX_CHART_DATE]: PlaceholderField,
-  [PROGRAM_DATA_ELEMENT_TYPES.COMPLEX_CHART_TYPE]: PlaceholderField,
-  [PROGRAM_DATA_ELEMENT_TYPES.COMPLEX_CHART_SUBTYPE]: PlaceholderField,
-};
-
-function getComponentForQuestionType(type) {
-  return QUESTION_COMPONENTS[type];
-}
+const Header = styled(Box)(({ theme }) => ({
+  borderBottom: `1px solid ${theme.palette.divider}`,
+}));
 
-const getTranslation = value => value;
+const Title = styled(Typography)(() => ({
+  fontSize: 16,
+  fontWeight: 500,
+  lineHeight: 2,
+}));
 
 export const SurveyView = () => {
-  const patient = useCurrentUser();
+  const { surveyId } = useParams<{ surveyId: string }>();
+  const { isPending, data: survey } = useSurveyQuery(surveyId);
+  const { additionalData, ...patient } = useCurrentUser();
+  const currentUser = {} as User;
+  const encounterType = ENCOUNTER_TYPES.CLINIC;
+
+  const onSubmit = async () => {};
+  const onCancel = async () => {};
+
+  if (isPending || !survey) {
+    return 'loading...';
+  }
+
   return (
-    <Card sx={{ width: '425px' }}>
-      {/*<TranslationContext.Provider value={{ getTranslation }}>*/}
-      {/*  <SettingsContext.Provider value={{}}>*/}
-      {/*      {testData.map(component => {*/}
-      {/*        return (*/}
-      {/*          <SurveyQuestion*/}
-      {/*            key={component.id}*/}
-      {/*            component={component}*/}
-      {/*            patient={patient}*/}
-      {/*            inputRef={() => {}}*/}
-      {/*            encounterType={ENCOUNTER_TYPES.ADMISSION}*/}
-      {/*          />*/}
-      {/*        );*/}
-      {/*      })}*/}
-      {/*  </SettingsContext.Provider>*/}
-      {/*</TranslationContext.Provider>*/}
-    </Card>
+    <Container>
+      <Header sx={{ p: 2 }}>
+        <Title variant="h2">{survey.name}</Title>
+      </Header>
+      <Box sx={{ p: 2 }}>
```

Comment:
think you can do this with box, such a nit pick!
```suggestion
      <Box p={2}>
```